library/Freckle/App/Database.hs

```diff
@@ -137,11 +138,23 @@ runDB
   -> m a
 runDB action = do
   pool <- asks getSqlPool
+  gauge <- Stats.lookupGauge Stats.dbConnections
+  let
+    hooks = setAlterBackend defaultSqlPoolHooks $ wrapSqlBackend mempty
+    hooks' = hooks
+      { runBefore = \conn mi -> do
+          Stats.incGauge gauge
+          runBefore hooks conn mi
+      , runAfter = \conn mi -> do
+          Stats.decGauge gauge
+          runAfter hooks conn mi
+      , runOnException = \conn mi e -> do
+          Stats.decGauge gauge
+          runOnException hooks conn mi e
+      }
   Stats.withGauge Stats.dbConnections $
```

Comment:
Wait, you have to remove this one or we're double-counting, no?

Reply:
Oh yeah, I wasn't done, basically I knew that there was another metric I wanted to capture in that spot, so I kept that code there.